google/cloud/opentelemetry/trace_exporter.h

```diff
@@ -29,7 +29,7 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
 // Make an OpenTelemetry Trace Exporter for Google Cloud Trace.
```

Comment:
For a future PR: fix the documentation.  This is supposed to be a public function.

Reply:
Ack.